src/Symfony/Component/Yaml/Parser.php

```diff
@@ -144,6 +144,10 @@ public function parse($value, $flags = 0)
                     $values['value'] = $matches['value'];
                 }
 
+                if (isset($values['value'][1]) && '?' === $values['value'][0] && ' ' === $values['value'][1]) {
+                    throw new ParseException('The parser does not support complex mappings.', $this->currentLineNb + 1, $this->currentLine);
```

Comment:
`Complex mappings are not supported.` ? To match the style of other error messages (e.g. `'Multiple documents are not supported.'`)